sansio_lsp_client/client.py

```diff
@@ -190,12 +385,15 @@ def recv(self, data: bytes) -> t.List[Event]:
 
         events: t.List[Event] = []
         for message in messages:
-            if isinstance(message, Response):
-                events.append(self._handle_response(message))
-            elif isinstance(message, Request):
-                events.append(self._handle_request(message))
-            else:
-                raise RuntimeError("nobody will ever see this, i hope")
+            try:
+                if isinstance(message, Response):
+                    events.append(self._handle_response(message))
+                elif isinstance(message, Request):
+                    events.append(self._handle_request(message))
+                else:
+                    raise RuntimeError("nobody will ever see this, i hope")
+            except:
```

Comment:
We don't want to catch `KeyboardInterrupt`, for example:

```suggestion
            except Exception:
```